lib/create-webpack-config-base.js

```diff
@@ -61,8 +61,14 @@ function createWebpackConfigBase(batfishConfig) {
         }
       ]
     ].concat(batfishConfig.babelPlugins || []);
-
-    if (!batfishConfig.production) {
+    if (batfishConfig.production) {
+      babelPlugins.push([
+        'transform-react-remove-prop-types',
+        {
+          ignoreFilenames: ['node_modules']
```

Comment:
That sounds reasonable! ignoreFilenames was the only option I thought might be relevant for this plugin, but it's way better to be safe and improve performance. Changes are in this commit f0c5fe9.